src/wallets/injected/SenderWallet.ts

```diff
@@ -97,16 +97,8 @@ class SenderWallet extends InjectedWallet implements ISenderWallet {
   }
 
   disconnect() {
-    return this.wallet.signOut()
-      .then((res) => {
-        if (res.result !== "success") {
-          throw new Error("Failed to sign out");
-        }
-
-        EventHandler.callEventHandler("disconnect");
-
-        return;
-      });
+    this.emitter.emit("disconnect", {});
+    return window[this.injectedGlobal].signOut();
```

Comment:
Looks like we've regressed on a merge conflict you had regarding my PR that went into dev this morning

Reply:
We had the following code before. You'll have hit a conflict and maybe used all of your version:
```ts
return this.wallet.signOut()
      .then((res) => {
        if (res.result !== "success") {
          throw new Error("Failed to sign out");
        }

        EventHandler.callEventHandler("disconnect");

        return;
      });

```